pkgs/applications/misc/rofimoji/default.nix

```
@@ -0,0 +1,46 @@
+{ buildPythonApplication
+, fetchFromGitHub
+, lib
+
+, waylandSupport ? true
+, x11Support ? true
+
+, ConfigArgParse
+, pyxdg
+, rofi
+, wl-clipboard
+, wtype
+, xdotool
+, xsel
+}:
+
+buildPythonApplication rec {
+  pname = "rofimoji";
+  version = "4.3.0";
+
+  src = fetchFromGitHub {
+    owner = "fdw";
+    repo = "rofimoji";
+    rev = version;
+    sha256 = "08ayndpifr04njpijc5n5ii5nvibfpab39p6ngyyj0pb43792a8j";
+  };
+
+  propagatedBuildInputs = with lib; [ ConfigArgParse pyxdg rofi ]
+    ++ ifEnable waylandSupport [ wl-clipboard wtype ]
+    ++ ifEnable x11Support [ xdotool xsel ];
+
+  # The 'extractors' sub-module is used for development
+  # and has additional dependencies.
+  preConfigurePhases = [ "postUnpack" ];
+  postUnpack = ''
+    rm -rf extractors
+  '';
+
```

This looks like patching the unpack'd source. You could probably just only do
```suggestion
  # The 'extractors' sub-module is used for development
  # and has additional dependencies.
  preConfigurePhases = [ "postUnpack" ];
  postPatch = ''
    rm -rf extractors
  '';

```

---

Good catch! Fixed.